README.md

```diff
@@ -34,128 +36,207 @@ git clone https://github.com/gnosis/dx-cli.git dx-cli
 cd dx-cli
 ```
 
-**3. Create `local.conf` using [local.conf.example](./local.conf.example)**
-> This step can be omitted if we plan to use the CLI for read-only operations.
+**2. Create `local.conf` using [local.conf.example](./local.conf.example)**
+> This step can be omitted if you plan to use the CLI for read-only operations.
+
+ **IMPORTANT**: Your mnemonic phrase will be required in order to link your wallet to the DutchX. Make sure you don't share
+ this information and that you keep the `local.conf` file that you will create offline. If you don't do this, you risk losing your funds! Never commit this to Github. If you don't know how to get your mnemonic phrase, you probably should not continue.
 
-Create a duplicate [local.conf.example](./local.conf.example) and call the new
+Go to the folder where you downloaded the repository, create a copy of the [local.conf.example](./local.conf.example) file
+ and call the new
 file `local.conf`.
 
 Edit the file in order to add your own secret mnemonic that will be used to sign
 the transactions.
 
-Aditionally, in this file you can add any custom configuration. Note that by 
-modifying only this file you can keep your CLI in sync with future changes.
-
 > **NOTE**: The `local.conf` is git ignored, so you can add your wallet config
 > here.
 
-> **IMPORTANT**: Never share this file or the mnemonic with anyone, the mnemonic
-> is protecting your funds from your account, so anyone with access to it could
-> take them.
 
-**4. Make sure the scripts are executable**
+
+**3. Make sure the scripts are executable**
 ```bash
 # Allow the CLI script to be executed
 chmod +x dutchx*
 ```
 
-**5. Network info: Review the list of tokens you want to use**
+**4. Network info: Review the list of tokens you want to use**
+
+> **NOTE**: This step can be skipped if you don't intend to list new tokens on the DutchX.
```

Comment:
Is not just if you don't want to list them. I don't plan to add every token. Anyways, we can leave it like this, and let see how this evolves